app/models/post.rb

```diff
@@ -0,0 +1,5 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: :User
+  has_many :comments
+  has_many :likes
+end
```

Comment:
- [ ]  Kindly add a method in Post model that that updates the posts counter for a user.
- [ ]  Kindly add a method in Post model that that returns the 5 most recent comments for a given post.